web/pandas/community/ecosystem.md

```diff
@@ -388,3 +388,65 @@ authors to coordinate on the namespace.
   | [pint-pandas](https://github.com/hgrecco/pint-pandas)                | `pint`     | `Series`, `DataFrame` |
   | [composeml](https://github.com/alteryx/compose)                      | `slice`    | `DataFrame`           |
   | [woodwork](https://github.com/alteryx/woodwork)                      | `slice`    | `Series`, `DataFrame` |
+
```

Comment:
umm what is this?

Reply:
If you mean the empty line I removed it. I'm using Unix line endings.